lua/lint/linters/dotenv_linter.lua

```diff
@@ -0,0 +1,22 @@
+return {
+  cmd = function()
+    local local_executable = vim.fn.fnamemodify("./bin/dotenv-linter", ":p")
+    local stat = vim.loop.fs_stat(local_executable)
+    if stat then
+      return local_executable
+    end
+    return "dotenv-linter"
+  end,
```

Comment:
This looks rather custom, or is this a established convention for `dotenv-linter` ?

Reply:
ok, done :)